packages/nextjs/src/config/webpack.ts

```diff
@@ -435,6 +436,33 @@ async function addSentryToClientEntryProperty(
   return newEntryProperty;
 }
 
+/**
+ * Make sure the instrumentation file has a `onRequestError` Handler
+ *
+ * @param projectDir The root directory of the project, where config files would be located
+ */
+function warnAboutMissingonRequestErrorHandler(projectDir: string): void {
+  const instrumentationPaths = [
+    ['src', 'instrumentation.ts'],
+    ['src', 'instrumentation.js'],
+    ['instrumentation.ts'],
+    ['instrumentation.js'],
+  ];
+  function hasOnRequestErrorHandler(pathSegments: string[]): boolean {
+    const filePath = path.resolve(projectDir, ...pathSegments);
+    try {
+      const content = fs.readFileSync(filePath, 'utf8');
+      return content.includes('export const onRequestError');
```

Comment:
`export const onRequestError` is definitely too strict. People could do `export function`, `export { onRequestError }` etc.

Let's just check for the presence of `"onRequestError"`.